digdag-standards/src/main/java/io/digdag/standards/operator/td/TDOperator.java

```diff
@@ -7,23 +7,32 @@
 import com.treasuredata.client.TDClientHttpConflictException;
 import com.treasuredata.client.TDClientHttpNotFoundException;
 import com.treasuredata.client.TDClientHttpUnauthorizedException;
-import com.treasuredata.client.model.TDExportJobRequest;
+import com.treasuredata.client.model.TDJob;
 import com.treasuredata.client.model.TDJobRequest;
-import com.treasuredata.client.model.TDSavedQueryStartRequest;
+import com.treasuredata.client.model.TDJobSummary;
 import io.digdag.client.config.Config;
+import io.digdag.client.config.ConfigElement;
 import io.digdag.client.config.ConfigException;
+import io.digdag.spi.TaskExecutionException;
 import io.digdag.util.RetryExecutor;
 import io.digdag.util.RetryExecutor.RetryGiveupException;
 
 import java.io.Closeable;
-import java.time.Instant;
-import java.util.Date;
+import java.util.UUID;
 
+import static com.treasuredata.client.model.TDJob.Status.SUCCESS;
 import static io.digdag.util.RetryExecutor.retryExecutor;
 
 public class TDOperator
         implements Closeable
 {
+    private static final String JOB_ID = "job_id";
```

Comment:
The original key names and https://github.com/treasure-data/digdag/pull/201/files#diff-3ade81f31ae01cbc057f3bc7abbc6b21R14 are lower camel cases. We'd better use an unified naming convention. Also, this change could cause an issue if remaining tasks stores the original keys and then tries to use them after the deployment.